boefjes/boefjes/katalogus/models.py

```diff
@@ -39,6 +39,7 @@ class Boefje(Plugin):
     scan_level: int = 1
     consumes: Set[str] = Field(default_factory=set)
     produces: List[str] = Field(default_factory=list)
+    mime_types: List[str] = Field(default_factory=list)
```

Comment:
I think this should be a set, as the elements are unique and order doesn't matter